fullstop/src/main/resources/db/migration/V6__create_outdated_ami_type.sql

```diff
@@ -0,0 +1,20 @@
+INSERT INTO fullstop_data.violation_type (
```

Comment:
please move the entire db/migration folder to a common library. This way we can enable Flyway migration both for fullstop and for fullstop-jobs.